src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -208,14 +208,21 @@ public function removedScopes()
      * @param  mixed  $id
      * @return $this
      */
-    public function whereKey($id)
+    public function whereKey($id, $negation = false)
```

Comment:
This is a breaking change to the method signature.

Reply:
Do you mean this does not follow the semantics of the method?
Maybe we can rename the "negation" to "reverse" to clarify what it does?

Reply:
A method signature cannot change in a stable release. Everyone overwriting this method will see their app break.